### PR TITLE
better ext handling

### PR DESCRIPTION
**Pull request by runk**

### CODE REVIEW

**Comment by camjc** [comment]
LGTM.

File: index.js [hunk 1]
```diff
@@ -22,31 +23,26 @@ module.exports = function(opts) {
...
 
-    // Handle only .jade files
-    if (opts.ext.indexOf(path.extname(filepath)) === -1)
-      return next();
+    if (!filepath)
+      return res.send(404)
 
     if (filepath.indexOf(opts.baseDir) !== 0)
-      return next(new Error('Invalid path'));
+      return res.send(403);
```
Comment (omgaz) [inline index.js]: Sure you mean 403?

Comment (omgaz) [inline index.js]: Nothing is testing this.

**Comment by omgaz** [comment]
LGTM

File: index.js (hunk 1 above)
Comment (runk) [inline index.js]: Yeah.. I'm a bit lazy :) Not sure if we able to even test.. I guess `url.parse` excapes rubbish like `..` etc.

**Pull request merged**